docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -4401,6 +4401,42 @@ Content-Type: text/plain; charset=utf-8
 -   **404** - plugin not installed
 -   **500** - plugin is active
 
+### Create a plugin
+
+`POST /plugins/(plugin name)/create`
```

Comment:
should this be `POST /plugins/create?name=<plugin name>` instead? Feels odd to have an endpoint named after a not-yet existing resource.

/cc @bfirsh